hashintel/hash
# Remove global styling in frontend

## 🌟 What is the purpose of this PR?




The HASH team is going to build a new version of the frontend in MUI, with improved designs. In a move towards that, and to assist in the styling migration, this PR removes the global styles and Tailwind preflights, similar to https://github.com/blockprotocol/blockprotocol/pull/245 in the blockprotocol repo.

As the Tailwind preflights were disabled already, the primary focus of this PR is to:
1. Delete the `Home.module.css` file, which was unused.
2. [Update `globals.scss`](https://github.com/hashintel/hash/pull/368/files#diff-4ce39571a477789ed1dea7768f3a357e23abd42e9fec0ccd05c19e2fdcb6e33eR1) to make the anchor and header styles specific to main, and remove any other resets.

## ❓ How to test this?

- Checkout this branch locally.
- Host the dev version of the repo.
- Check whether there are any major styling differences within the pages.


---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1201905971519552
